core/access_api.php

```diff
@@ -836,3 +836,26 @@ function access_threshold_min_level( $p_threshold ) {
 	}
 
 }
+
+/**
+ * Checks if the user can view the handler for the bug.
+ * @param BugData      $p_bug     Bug to check access against.
+ * @param integer|null $p_user_id Integer representing user id, defaults to null to use current user.
+ * @return boolean whether user can view the handler user.
+ */
+function access_can_see_handler_for_bug( BugData $p_bug, $p_user_id = null ) {
```

Comment:
Typically our APIs take in an issue ID instead of a BugData.  Calling the bug_get_field() API internally should hit the cache.

Reply:
I followed these:
access_can_close_bug()
access_can_reopen_bug()
which also have the object as parameter.

Note that the 3 usages are already using the object.

This was Damien comment: 8407d61091677038373875f2dd7eb2540b89f4ab
> use a BugData object instead of a bug id as parameter for
    several functions in access_api.php and html_api.php, resulting in
    simpler and more efficient code thanks to fewer calls to bug_get() and
    bug_get_field().

Reply:
ok